Source/CarShack/Controllers/Cars/CarsController.cs

```diff
@@ -2,9 +2,9 @@
 using CarShack.Hypermedia.Cars;
 using CarShack.Util;
 using Microsoft.AspNetCore.Mvc;
-using WebApi.HypermediaExtensions.Exceptions;
-using WebApi.HypermediaExtensions.WebApi.AttributedRoutes;
-using WebApi.HypermediaExtensions.WebApi.ExtensionMethods;
+using RESTyard.WebApi.Extensions.Exceptions;
```

Comment:
IMHO the namespace should be RESTyard.AspNetCore.Extensions.XXX 
Web API is not the product this package / code is for.